### PR TITLE
Fix of issue number 30

### PR DESCRIPTION
**Pull request by HasibulKabir**
maybe there can be a bug. check that before Marge.

### CODE REVIEW

File: plugins/youtube_dl_button.py [hunk 1]
```diff
@@ -109,6 +109,21 @@
     tmp_directory_for_each_user = Config.DOWNLOAD_LOCATION + "/" + str(update.from_user.id)
     if not os.path.isdir(tmp_directory_for_each_user):
         os.makedirs(tmp_directory_for_each_user)
+    if len(custom_file_name) > 64:
+        await bot.edit_message_text(
+            text="File Name is greater than 64 characters. Reducing File name to 64 charterers.",
+            chat_id=update.message.chat.id,
+            message_id=update.message.message_id
+        )
+        file_exten = custom_file_name.split(".")
```
Comment (SpEcHiDe) [inline plugins/youtube_dl_button.py]: can you add rsplit here? 

consider a situation where file name has multiple . DOTs? 😅

**Comment by SpEcHiDe** [comment]
good. But we don't yet know if 64 is really the limit. 😅😅
cc: #30

**Comment by HasibulKabir** [comment]
ok. close pull requests

**Comment by SpEcHiDe** [comment]
please wait.. let me confirm with Telegram.dog/BotSupport

**Pull request closed without merging**